cmake/Modules/FindQwt.cmake

```diff
@@ -15,10 +15,10 @@ find_path(QWT_INCLUDE_DIRS
   ${CMAKE_INSTALL_PREFIX}/include/qwt
   ${CMAKE_PREFIX_PATH}/include/qwt
   PATHS
+  /usr/local/include/qt5/qwt6
   /usr/local/include/qwt-${QWT_QT_VERSION}
   /usr/local/include/qwt
   /usr/include/qwt6
-  /usr/include/qt5/qwt
```

Comment:
If you are only adding, why did you remove here?

Reply:
Because I screwed up. Sorry :-(